variables.tf

```diff
@@ -67,6 +67,17 @@ variable "create_cos_bucket" {
   default     = true
 }
 
+variable "cross_region_location" {
+  description = "Specify the cross-regional bucket location. Supported values are 'us', 'eu', and 'ap'."
```

Comment:
Specify the cross-regional bucket location. Supported values are 'us', 'eu', and 'ap'. If you pass a value for this, do not pass one for var.region